openlibrary/i18n/__init__.py

```diff
@@ -37,14 +37,20 @@ def _validate_catalog(catalog, locale):
     for message in catalog:
         message_errors = validate(message, catalog)
 
-        if message_errors:
-            if message.lineno:
+        if message.lineno:
+            if message.fuzzy:
+                print(
+                    f'openlibrary/i18n/{locale}/messages.po:'
+                    f'{message.lineno}: {message.string}'
```

Comment:
```suggestion
                    f'{message.lineno}: "{message.string}" is fuzzy'
```
Output is a little confusing without some indication that these strings are fuzzy.